### PR TITLE
Copter: make LOITER_TIME send "Reached command" message to GCS

### LINKED ISSUE #10619: Copter: LOITER_TIME command does not send "Reached command" message to GCS
Title pretty much says it all - the LOITER_TIME mission command does not seem to be sending a "Reached command" message to the GCS.

### PR DESCRIPTION
**Pull request by arjun7965**
fixes #10619

### CODE REVIEW

File: ArduCopter/mode_auto.cpp [hunk 1]
```diff
@@ -1669,7 +1669,13 @@ bool Copter::ModeAuto::verify_loiter_time()
     }
 
     // check if loiter timer has run out
-    return (((millis() - loiter_time) / 1000) >= loiter_time_max);
+    if (((millis() - loiter_time) / 1000) >= loiter_time_max) {
+        gcs().send_text(MAV_SEVERITY_INFO, "Reached command #%i",cmd.index);
+        return true;
+    } else {
```
Comment (peterbarker) [inline ArduCopter/mode_auto.cpp]: I'd typically do an early-return instead if the time hasn't been reached.

Comment (arjun7965) [reply ArduCopter/mode_auto.cpp]: Done. Spline, regular waypoint has it that way. Do I submit a PR for that?

Comment (arjun7965) [reply ArduCopter/mode_auto.cpp]: Submitted a PR #10630

**Comment by rmackay9** [comment]
Merged, thanks!

**Pull request merged**